docs/deploying.md

```diff
@@ -31,64 +26,126 @@ image. You push an image to, and then pull the same image from, this registry.
 This a good exercise for understanding the basic interactions a client has with
 a local registry.
 
-1. Install Docker.
+### Setting up with Docker Machine
```

Comment:
Agreed with that.
Large part of the confusion arise from third-party tools that people mess-up with.

The basic stance the registry doc should adopt is: one line docker commands to setup.

I do believe compose is a good choice for managing complex setups, but anything else (basic linux installation, or setting-up docker itself) should really live where they belong, which will also reduce maintenance costs over duplicated information.